Jenkinsfile

```diff
@@ -27,7 +27,7 @@ pipeline {
                     cd ..
                     # Workaround for "unflattened" file, possibly due to matrix
                     find . -name version.properties -exec cp {} . \\;
-                    test -e version.properties
+                    test -e version.properties || touch version.properties
```

Comment:
I ran into this in py3-ci, but touching ended up not being enough in my particular case. Instead I needed to hard-code the versions. If you run into the same problem, we may need to rethink (e.g. if a version is not set, don't try to replace it)